tests-e2e/ocw-ci-test-course/shortcodes.spec.ts

```diff
@@ -37,3 +37,68 @@ test("Related resources link to correct page", async ({ page }) => {
   const href = await resourceLink.getAttribute("href")
   expect(href).toBe("/courses/ocw-ci-test-course/resources/example_pdf/")
 })
+
+test("Start and end times does not exist", async ({ page }) => {
+  const course = new CoursePage(page, "course")
+  await course.goto("resources/ocw_test_course_mit8_01f16_l01v02_360p")
+  const src = await page.locator("iframe.vjs-tech").getAttribute("src")
+
+  expect(src).not.toMatch(/.*?start=.*/)
+  expect(src).not.toMatch(/.*?end=.*/)
+})
+
+test("Start time exists", async ({ page }) => {
+  const course = new CoursePage(page, "course")
+  await course.goto("resources/ocw_test_course_mit8_01f16_l01v01_360p")
+  const src = await page.locator("iframe.vjs-tech").getAttribute("src")
+  expect(src).toMatch(/.*?start=13.*/)
+})
+
+test("End time exists", async ({ page }) => {
+  const course = new CoursePage(page, "course")
+  await course.goto("resources/ocw_test_course_mit8_01f16_l01v01_360p")
+  const src = await page.locator("iframe.vjs-tech").getAttribute("src")
+  expect(src).toMatch(/.*?end=50.*/)
+})
+
+test("Start and end time exists", async ({ page }) => {
+  const course = new CoursePage(page, "course")
+  const expectedStartTime = "13"
+  const expectedEndTime = "50"
+
+  await course.goto("resources/ocw_test_course_mit8_01f16_l01v01_360p")
+  const src = await page.locator("iframe.vjs-tech").getAttribute("src")
+
+  const urlParams = new URLSearchParams(src || "")
+  expect(urlParams.get("start")).toEqual(expectedStartTime)
+  expect(urlParams.get("end")).toEqual(expectedEndTime)
+})
+
+test("Transcripts start time matches video start time", async ({ page }) => {
+  const course = new CoursePage(page, "course")
+  const videoSection = new VideoSection(page)
+  const expectedStartTime = "13"
+
+  await course.goto("resources/ocw_test_course_mit8_01f16_l01v01_360p")
+  const src = await page.locator("iframe.vjs-tech").getAttribute("src")
```

Comment:
```suggestion
  const src = await videoSection.iframe().getAttribute("src")
```

Nicer to not use the `.vjs-tech` class. *There's still a class in the `VideoSection` class used for scoping the locators, but that CSS class is ours, and if it changes, we only need to change it in one spot, namely VideoSection.*

IMO, worth making this change in the other tests, too.